### ISSUE TITLE
destabalizes the quantity of

### ISSUE THREAD

**Issue by ValWood** [oborel/obo-relations#395]
definition
An entity A directly interacts with B and A has an activity that decreases the amount of an entity B by degradating it.

1) degradating  -> degrading.

2. We have continual issues with the use of "stability" and "stabilization" in GO. These are more the "outcome"  than the relationship or process but have become conflated. 
To prevent the percolating into RO too I suggest perhaps:

directly negatively regulates quantity of, by degradation (or similar)

as the primary name and making "destabilizes the quantity of" synonymous

Also, this would match the definition better (stability is the balance between production and degradation)

**Comment by ValWood** [comment]
for
stabilizes quantity of

definition
An entity A physically interacts with B and A has an activity that increases the amount of an entity B by stabilizing it.

the definition is slightly different. It doesn't mention degradation, but it's also a bit circular.

I suggest mirroring the 'destabilizes' term and describing in terms of degradation, because  the corresponding terms for increase and decrease of production already exist.

**Comment by nlharris** [comment]
Would this be a case where Val should make a PR and request reviews?

**Comment by pgaudet** [comment]
@ValWood How would you use that, as an extension ? 
What is the GO term you would use, 'regulation of x stability' - such as GO:0043487? 
why not use 'has input' like we do for most other terms ?

**Comment by ValWood** [comment]
Don't worry about this. It's one of the comments I made when I was a thesis opponent and I needed to look at  RO. 

I still think the definition is incorrect for the term. 

"destabilizes the quantity of"is not trrreally the right label for 'degradation'
An entity A directly interacts with B and A has an activity that decreases the amount of an entity B by degradating it.

But this isn't a GO issue, it was only an observation I made in passing.
